migrator/migrations/m_192_to_m_193_report_config_migrations/migration_impl.go

```diff
@@ -0,0 +1,172 @@
+package m192tom193
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_192_to_m_193_report_config_migrations/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/pkg/uuid"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+func migrate(database *types.Databases) error {
+	db := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, db, updatedSchema.CreateTableReportConfigurationsStmt)
+	db = db.WithContext(database.DBCtx).Table("report_configurations")
+
+	dbSnapshot := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, dbSnapshot, updatedSchema.CreateTableReportSnapshotsStmt)
+	dbSnapshot = dbSnapshot.WithContext(database.DBCtx).Table("report_snapshots")
+
+	dbNotifier := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, dbNotifier, updatedSchema.CreateTableReportConfigurationsStmt.Children[0])
+	dbNotifier = dbNotifier.WithContext(database.DBCtx).Table("report_configurations_notifiers")
+	rows, err := db.Rows()
+	if err != nil {
+		return errors.Wrapf(err, "failed to iterate table %s", "report_configurations")
+	}
+	if rows.Err() != nil {
+		utils.Should(rows.Err())
+		return errors.Wrapf(rows.Err(), "failed to get rows for %s", "report_configurations")
+	}
+	defer func() { _ = rows.Close() }()
+	var convertedReportConfigs []*updatedSchema.ReportConfigurations
+	var reportSnapshots []*updatedSchema.ReportSnapshots
+	var reportNotifiers []*updatedSchema.ReportConfigurationsNotifiers
+	for rows.Next() {
```

Comment:
This is a lot of stuff in one loop.  We should probably break some of this up into functions and such.